media/js/src/simulationTwo/simulationTwo.jsx

```diff
@@ -62,6 +49,46 @@ export const SimulationTwo = () => {
         setControls({});
     }, [data]);
 
+    useEffect(() => {
+        console.log('isCorrect:', isCorrect);
+        if (choice && !Object.values(isCorrect).includes(false)) {
+            setIsComplete({...isComplete, [choice]: true});
+            setData(null);
+            setIsSubmitted(false);
+        }
+    }, [isCorrect]);
+
+    const takeaways = {
```

Comment:
One of the asks is to make it so that Zarina can modify the text for the questions and options. I've rearranged things on my end for that purpose so that she can use jsx. Just make sure she's able to do that here as well.

Reply:
I spoke with Zarina, she said not to change anything. But if Seyhan wants Katex or html formatting in her question options I will have to tweak it. But that can be done in another (smaller) PR.